OpenEdXMobile/src/main/java/org/edx/mobile/deeplink/DeepLinkManager.java

```diff
@@ -7,34 +7,26 @@
 import org.edx.mobile.module.prefs.LoginPrefs;
 import org.edx.mobile.util.Config;
 import org.edx.mobile.view.Router;
-import org.json.JSONException;
-import org.json.JSONObject;
 
 /**
  * Utility class to handle the navigation within the app through deep links.
  */
 public class DeepLinkManager {
-    public static final String KEY_CLICKED_BRANCH_LINK = "+clicked_branch_link";
     protected static final Logger logger = new Logger(DeepLinkManager.class);
-    private static final String KEY_SCREEN_NAME = "screen_name";
-    private static final String KEY_COURSE_ID = "course_id";
-    private static final String KEY_TOPIC_ID = "topic_id";
-    private static final String KEY_THREAD_ID = "thread_id";
-    private static final String KEY_PATH_ID = "path_id";
 
-    public static void parseAndReact(@NonNull Activity activity, @NonNull JSONObject paramsJson) throws JSONException {
-        logger.debug("DeepLink received. JSON Details:\n" + paramsJson);
+    public static void onDeepLinkReceived(@NonNull Activity activity, @NonNull final DeepLink deepLink) {
```

Comment:
We have to trigger some logs, in a case when app receiving invalid/not implemented `screen_name` (e.g app receive `pro` screen name instated of `profile`)